v3/docs/01-getting-started/a-overview/index.mdx

```diff
@@ -1,66 +1,61 @@
-   [Build a dApp](/tutorials/v3/proof-of-existence) and [Add a Pallet](/tutorials/v3/add-a-pallet).
-
-3. **With Substrate Core**: The entire FRAME system can be ignored, and the runtime can be designed and implemented from
-   scratch. This could be done in _any language_ that can target [WebAssembly](https://webassembly.org/). If the runtime
-   can be made to be compatible with the abstract block authoring logic of the Substrate node, then you can simply
-   construct a new genesis block from your Wasm blob and launch your chain with the existing Rust-based Substrate
-   client. If not, then you will need to alter the client's block authoring logic, and potentially even alter the header
-   and block serialization formats. In terms of development effort, this is by far the most difficult way to use
-   Substrate, but also gives you the most freedom to innovate.
-
-Substrate allows developers to make choices between technical freedom and ease of development in every step of their design
-decisions. The diagram below illustrates the nature of this flexibility.
+To create a custom blockchain, let’s understand the Substrate framework and other elements. The basic unit where blockchain runs is a node. Since blockchain is a decentralized peer-to-peer system, this node runs inside a client and provides all basic components the system needs to function, such as p2p networking, storage, logic for block processing and consensus, and ability to interact with external blockchains.                          
+                          
+### Usage
 
+Substrate allows developers to make choices between technical freedom and ease of development in every step of their design process. The diagram below illustrates the nature of this flexibility (starting from bottom-up).
 ![Technical Freedom vs Development Ease](../../../../src/images/docs/getting-started/technical-freedom.png)
 
+#### 1. Substrate Node
+
+Substrate node is the highest level to start at, which provides most pre-built functionality [Substrate Node](https://github.com/paritytech/substrate/tree/master/bin/node) and includes all default implementations for basic components. You can customize or [configure](https://github.com/paritytech/substrate/blob/master/bin/node/cli/src/chain_spec.rs) the genesis block (that is the initial state) of the chain and run the node get familiar with Substrate. 
+For example, you can import a JSON file and launch your own blockchain. The JSON file allows you to configure the genesis state of the modules that compose the Substrate Node's runtime, such as: Balances, Staking, and Sudo. You can learn more about running a Substrate node in the Create Your First Substrate Chain and Start a Private Network tutorials.
+
+Once you are familiar and are ready to customize or build your own blockchain, you can go one step down and start at the Framework for Runtime Aggregation of Modularized Entities (FRAME).
+
```

Comment:
I like what you are trying to do here—going through the ease of development/technical freedom stack—I won't nit-pick the language. As Dan mentioned, we are doing some major renovation on the doc content.
For reference, here's what this section looks like in the in-progress rewrite:

# Substrate empowers builders

Substrate takes a modular and flexible approach to blockchain development. With every design decision, you can choose between the complexity of technical freedom and the ease of developing with predefined modules.

The following diagram illustrates the nature of this flexibility.

<image: Technical freedom vs development ease>

The main use cases for Substrate blockchains reflect this sliding scale between technical freedom and development ease. At one end of the spectrum, you can deploy predefined Substrate nodes with minimal configuration and launch a blockchain with virtually no development effort. At the other end of the spectrum, you can design and implement a Substrate-based blockchain from scratch, giving you the technical freedom to innovate.

Naturally, the most common use case falls between these two extremes. In the middle of the spectrum, you can use FRAME—an acronym for Framework for Runtime Aggregation of Modularized Entities—to create a customized Substrate runtime. With this approach, you can choose how much control you have over the blockchain logic by selecting and configuring the modules—called pallets—that you want to use from a library. If a pallet doesn't exist for the functionality you need, you can use FRAME to create your own custom pallet, then add it to your customized runtime.